components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/OAuth2Service.java

```diff
@@ -660,4 +661,26 @@ private String getErrorCode(OAuthClientAuthnContext oAuthClientAuthnContext) {
         return errorCode;
     }
 
+
+    /**
+     * Handles the authorization request denied by user.
+     *
+     * @param oAuth2Parameters OAuth parameters.
+     * @return OAuth2AuthorizeRespDTO Authorize Response Data Transfer Object.
+     */
+    public OAuth2AuthorizeRespDTO handleUserConsentDenial(OAuth2Parameters oAuth2Parameters) throws IdentityOAuth2Exception {
```

Comment:
Do we need return for the handle methods, 
It seems the returned value is not used

Reply:
It is because issue method returns. And in case if there is a need for any responseTypeHandlers wanted to update response error codes and return them [ though we don't need anything to return].